apps/wallet/src/ui/app/hooks/useGetCoinBalance.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type CoinBalance, type SuiAddress } from '@mysten/sui.js';
+import { useQuery, type UseQueryResult } from '@tanstack/react-query';
+
+import { useRpc } from '_hooks';
+type GetCoinBalanceProps = {
+    address: SuiAddress | null;
```

Comment:
Why not have these hooks grab the address itself? Not super opinionated in this, just curious.

Reply:
I think there may be instances once multi-account is active that we want the balance of the address other than the current/active one